### PR TITLE
PHP type parser method for PdbColumn

### PR DESCRIPTION
**Pull request by jamiemonksuk**

### CODE REVIEW

File: src/Models/PdbColumn.php [hunk 1]
```diff
@@ -74,4 +74,74 @@ public function check(array $tables, PdbTable $table): array
...
+     *          list ($key, $value) = Sprout::iterableFirst(['an' => 'array']);
+     */
+    public static function iterableFirst($iter)
+    {
+        foreach ($iter as $k => $v) {
+            return [$k, $v];
+        }
+
+        return null;
+    }
```
Comment (gwillz) [inline src/Models/PdbColumn.php:94]: This exists in the kb Arrays helper. Really don't want to start polluting these models with this kind of stuff.

File: src/Models/PdbColumn.php [hunk 2]
```diff
@@ -74,4 +74,74 @@ public function check(array $tables, PdbTable $table): array
...
+
+
+    /**
+     * Get the PHP var type of column based on the column data type
+     *
+     * @return string
+     */
+    public function getPhpType()
```
Comment (gwillz) [inline src/Models/PdbColumn.php:102]: This whole method exists (in a similar way) in PdbHelpers. I was hoping that would get updated instead and used here.

**Comment by jamiemonksuk** [comment]
I'll move some stuff around ^^

**Comment by jamiemonksuk** [comment]
Nevermind, I see you did!

**Pull request merged**